releases/unreleased/confluence-max_contents-per-query.yml

```diff
@@ -0,0 +1,9 @@
+---
+title: Confluence max_contents per query
+category: added
+author: Quan Zhou <quan@bitergia.com>
+issue: null
+notes: >
+    This allows setting a maximum number of contents per query.
+    By default, the number is 200, but this can be changed by
+    adding the `--max-contents` argument.
```

Comment:
```suggestion
    This parameter sets the maximum number of contents requested
    per query. By default, this number is 200, but this can be changed by
    adding the `--max-contents` argument. This will help to bypass those
    instances where the maximum limit is lower than Perceval's default
    value.
```